src/Symfony/Component/Serializer/Serializer.php

```diff
@@ -40,8 +40,6 @@ class Serializer implements SerializerInterface, NormalizerInterface, Denormaliz
     protected $encoder;
     protected $decoder;
     protected $normalizers = array();
-    protected $normalizerCache = array();
-    protected $denormalizerCache = array();
```

Comment:
I'd say we should keep them but never fill them, and keep reading from them in case someone fills them with its own logic